pkg/services/email.go

```diff
@@ -56,6 +56,7 @@ type EmailOptions struct {
 	Username           string `json:"username"`
 	Password           string `json:"password"`
 	From               string `json:"from"`
+	SendHtml           bool   `json:"sendHtml"`
```

Comment:
Can you change it to just `html` please? I think `send` is redundant.

Reply:
Thank you for reviewing.
I was thinking whether to use `html` or `sendHtml`, indeed it is redundant so I have changed to `html`.